From 6b88113ca7ae97752c87859bf199b4cbd29924f3 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@st.com>
Date: Wed, 23 Feb 2022 14:38:21 +0100
Subject: [PATCH 383/529] ARM: dts: stm32: remove alias ethernet0

No more needed, MAC address is retrieve with nvmem

Signed-off-by: Christophe Roullier <christophe.roullier@st.com>
Change-Id: Ibf373547da3d2058771228089165feb2260bd56a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/241017
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/284375
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157f-dk2.dts | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm/boot/dts/stm32mp157f-dk2.dts
+++ b/arch/arm/boot/dts/stm32mp157f-dk2.dts
@@ -19,7 +19,6 @@
 	compatible = "st,stm32mp157f-dk2", "st,stm32mp157";
 
 	aliases {
-		ethernet0 = &ethernet0;
 		serial3 = &usart2;
 	};
 
